main.tf

```diff
@@ -49,6 +49,7 @@ app_settings = {
     "PORT" = "8080"
     "SECRET_KEY" = "secret-key"
     "TRELLO_BOARD" = "60eacfe12a6b3c534957c408"
+    "LOG_LEVEL" = var.loglevel
```

Comment:
You've added LOG_LEVEL here, but it would be worthwhile also adding LOGGLY_TOKEN (though it is a bit more work since we'll need to set it as a variable, encrypt it for Travis and make sure Terraform picks it up!)